Updated the interfaces and implementation of the transformation strat… (#233)
EMMC-ASBL/oteapi-core: README.md

Standard transformation strategies: *celery/remote*

> The tranformation strategy has consolidated the execution of the

Reviewer: ```suggestion
The transformation strategy has consolidated the execution of the
```

transformation with the `get()` method to unify the strategy interfaces.
`get()` is intended to start an asynchronous process and return a

> *task_id* which can be queried using the `status()` method (outside of pipeline)

Reviewer: ```suggestion
*task_id* which can be queried using the `status()` method (outside of a pipeline).
```

Standard transformation strategies: *celery/remote*

The tranformation strategy has consolidated the execution of the

> transformation with the get() method to unify the strategy interfaces.
> get() is intended to start an asyncronous process and return a
> *tast_id* which can be queried using the status() method (outside of a

Reviewer: ```suggestion
transformation with the `get()` method to unify the strategy interfaces.
`get()` is intended to start an asynchronous process and return a
*task_id* which can be queried using the `status()` method (outside of a
```

I suppose we're not doing one-line-per-one-sentence in README files anymore (hate that), so I won't suggest changes for that...

#### `setup.py`

> In the package's `setup.py` file, one can specify entry points.

Reviewer: This change should not have happened. Did you run the latest `pre-commit`? The `trailing-whitespace` hook specifically has the `--markdown-linebreak-ext=md` option set to avoid it doing this...
```suggestion
In the package's `setup.py` file, one can specify entry points.  
```

> In the package's `setup.py` file, one can specify entry points.

Reviewer: ```suggestion
In the package's `setup.py` file, one can specify entry points.  
```
Again, this white space should **not** be removed. It's markdown-ish for (new line, but don't create a new section).
